### DISCUSSION TITLE
EL5 MyOpenMath

### DISCUSSION THREAD

**Discussion by tdegeorge** [TeamBasedInquiryLearning/precalculus#362] [General]
Reviewer: @AbbyANoble 

I feel like this section is pretty short....although some of the questions really have multiple questions in them. Do you think we need to add more?

Questions 1 - 3: recap of power, product, and quotient properties.
Questions 4 - 7: Expand using properties
Questions 8 - 14: Expand using properties
Question 15 - 19: Change of base

**Comment by AbbyANoble** [comment]
I think it is long enough! So many of them have multiple parts.
